plugins/aap-backend/config.d.ts

```diff
@@ -0,0 +1,18 @@
+import { TaskScheduleDefinitionConfig } from '@backstage/backend-tasks';
+
+interface AapConfig {
+  baseUrl: string;
+  /** @visibility secret */
+  authorization: string;
+  system?: string;
+  owner?: string;
+  schedule?: TaskScheduleDefinitionConfig;
+}
+
+export interface Config {
+  catalog?: {
+    providers?: {
+      aapResourceEntity?: { [key: string]: AapConfig };
```

Comment:
Can we rename this? If we ever want to extend this to something else than Resource, this name will become confusing. No other catalog provider is named like this.

Would a simple aap be enough? WDYT?

```suggestion
      aap?: { [key: string]: AapConfig };
```